_config.yml

```diff
@@ -122,4 +122,12 @@ workshop_sched_id: "ComputeCanada_Graham_slurm"
 # workshop_host_login: "login.cirrus.ac.uk"
 # workshop_host_location: "EPCC, The University of Edinburgh"
 # workshop_host_prompt: "[yourUsername@cirrus-login0 ~]$ "
-# workshop_host_homedir: "/lustre/home/tc001"
\ No newline at end of file
+# workshop_host_homedir: "/lustre/home/tc001"
+# Example: Myriad cluster at UCL, University College London
+# workshop_host: "Myriad"
+# workshop_host_id: "UCL_Myriad"
```

Comment:
I don't understand why the UCL parts of this file are commented out, and the University of Waterloo parts left active.

Reply:
That's the default of the lesson, when we merge it we can then activate so that we can contribute this to upstream.

Reply:
Yes but this is for our own repo. So our version should have us activated, surely? Or are you going to send the PR upstream as well?

Reply:
Our version will be activated, but we need to push this upstream too, that's why I'm separating these things so it's easier to handle.

Reply:
OK